### ISSUE TITLE
marginTop="3" (with quotes) stopped working

### ISSUE THREAD

**Issue by StokeMasterJack** [smyte/jsxstyle#95]
<Row marginTop="3"> used to work. Changing it to marginTop={3} fixed the problem. 

If using quotes is no longer supposed to work, it would be helpful to generate an error message instead of failing silently.

**Comment by meyer** [comment]
@StokeMasterJack jsxstyle uses the unitless prop conversion function from React. I suspect they changed quoted unitless prop behaviour between React 0.14 (the version we used to use) and React 16 (the version we use now). `marginTop="3"` shouldn’t have worked at all, and we probably should’ve showed some kind of warning if a string value was provided to a known unitless property.

I started a discussion about showing more helpful warning messages in development in #93. This would be one of them. If there’s anything else (frustrations or unexpected behaviour) you’ve run into, feel free to add them there.
